go/felix/rules/endpoints.go

```diff
@@ -45,6 +46,7 @@ func (r *DefaultRuleRenderer) HostDispatchChains(endpoints map[string]proto.Host
 	for ifaceName, _ := range endpoints {
 		names = append(names, ifaceName)
 	}
+	sort.Strings(names)
```

Comment:
I think this needs to be pushed down to avoid non-determinism in the workload case too (i.e. the mainline code relied on determinism here too).